device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubReceiveTask.java

```diff
@@ -3,36 +3,49 @@
 
 package com.microsoft.azure.sdk.iot.device.transport;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * Polls an IoT Hub for messages and invokes a callback if one is found.
  * Meant to be used with an executor that continuously calls run().
  */
 public final class IotHubReceiveTask implements Runnable
 {
     protected final IotHubTransport transport;
-
+    /**
+     * Private logger for class
+     */
+    final Logger logger = LoggerFactory.getLogger(IotHubReceiveTask.class);
+    		
```

Comment:
Can this be changed to the logger (CustomLogger) in use currently until we address proper logging method.